lua/nvim-treesitter/parsers.lua

```diff
@@ -1804,6 +1804,14 @@ list.rust = {
   maintainers = { "@amaanq" },
 }
 
+list.ralph = {
+  install_info = {
+    url = "https://github.com/alephium/tree-sitter-ralph",
+    files = { "src/parser.c" },
+  },
+  maintainers = { "@tdroxler" },
+}
+
```

Comment:
put it in its correct spot alphabetically - below racket and above rasi

Reply:
also in the readme and the lockfile

Reply:
oops...Done

Reply:
and the lockfile ;)

Reply:
-_-'

Reply:
done